### DISCUSSION TITLE
Problemas conexión Lab 04.01 - 04.02

### DISCUSSION THREAD

**Discussion by joanfco30** [rramosp/ai4eng.v1#115] [04 - GESTIÓN DE DATOS]
Buenos noches he tenido problemas para conectarme al laboratorio numero 4. 
Puedo ejecutar todo el notebook pero tengo problemas para evaluar: Al abrir el link para solicitar la contraseña, debo abrir un link pero me sale el siguiente problema: 

Les ha pasado ? agradecería como se soluciona para dar continuidad a las actividades. 
Muchas gracias

![image](https://user-images.githubusercontent.com/60493083/160308138-91daaa83-01ab-4e04-a788-c3c86643840a.png)

![image](https://user-images.githubusercontent.com/60493083/160308072-4db68bac-43eb-4358-a17f-3271197bc2a2.png)

**Comment by ldavidflorez** [comment]
Estoy teniendo el mismo problema.

**Comment by Felipe-RA** [comment]
Saludos

Recientes cambios en Google Auth están generando problemas con los Login, estamos trabajando para solucionarlo lo más pronto posible

Les estaremos informando cualquier novedad

Hasta pronto
